pyquil/api/_qpu.py

```diff
@@ -28,22 +27,23 @@
 from pyquil.quilatom import (
     MemoryReference,
 )
-import qcs_sdk
+from qcs_sdk import QCSClient
+from qcs_sdk.qpu.api import submit, retrieve_results, ExecutionResult
+from qcs_sdk.qpu.rewrite_arithmetic import build_patch_values
 
 
-def decode_buffer(buffer: "qcs_sdk.ExecutionResult") -> np.ndarray:
+def decode_buffer(buffer: ExecutionResult) -> NDArray[Union[np.complex64, np.int32]]:
```

Comment:
Does this type hint imply that the returned ndarray can have contain a mix of complex and integer values. Should the type hint be something like:

`Union[NDArray[np.complex64], NDArray[np.int32]]`

Reply:
You are correct and this is fixed 👍